packages/forms/src/UIForm/fields/Datalist/Datalist.component.js

```diff
@@ -1,13 +1,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
-import keycode from 'keycode';
-import get from 'lodash/get';
-
-import Typeahead from '@talend/react-components/lib/Typeahead';
-
+import DataList from '@talend/react-components/lib/Datalist';
```

Comment:
Rename into `DataListComponent` to not interfere with this file form component name.
Don't forget to change it in jsx